python/singa/autograd.py

```diff
@@ -77,6 +82,156 @@ def gradients(y, dy=None):
     return grads
 
 
+
+def get_onnx_model(y, dy=None):
+    ######################
+    ##onnx
```

Comment:
can you add some comments to this function?
it's difficult to understand the workflow without any description.

Reply:
Yes, I add some comments.
the main part of the function is autograd.backward.
find each node of the computational graph and save to onnx model